openpype/hosts/photoshop/plugins/publish/collect_color_coded_instances.py

```diff
@@ -32,7 +32,7 @@ class CollectColorCodedInstances(pyblish.api.ContextPlugin):
     # TODO check if could be set globally, probably doesn't make sense when
     # flattened template cannot
     subset_template_name = ""
-    create_flatten_image = False
+    create_flatten_image = "no"
```

Comment:
I would add some explanation about possible values and what is the difference especially what is difference in `yes` and `only`. Maybe I would change the value to tell what only e.g. `flatten_only`.

Reply:
Added more docstrings.
In setting it is actually as enum

Reply:
> In setting it is actually as enum

I know but it doesn't meant the value can't tell what it means.

As I'm reading the docstrings wouldn't be better to have "Flatten only (flatten_only)" and "Flatten with image (flatten_with_image)". Because what you have now as flatten only more suits value yes, but current yes does more then flatten.

Reply:
Ok, makes sense. Updated.